packages/lib/src/components/Card/components/CardInput/CardInput.tsx

```diff
@@ -75,6 +76,8 @@ const CardInput: FunctionalComponent<CardInputProps> = props => {
     const showBillingAddress = props.billingAddressMode !== AddressModeOptions.none && props.billingAddressRequired;
 
     const partialAddressSchema = handlePartialAddressMode(props.billingAddressMode);
+    // Keeps the value of the country set initially by the merchant, before the Address Component mutates it
+    const partialAddressCountry = useRef<string>(partialAddressSchema && props.data?.billingAddress?.country);
```

Comment:
I think we should default to `US` in case of partial address mode. This rule was designed for `US` merchants in mind and I think this makes the change safer.

Reply:
Not really - if you check the scope of the internal ticket, it was reported by one merchant from Brazil. In the comments you can also see that, if no country is set, then we don't apply any validation.